resources/javascripts/script.js

```diff
@@ -0,0 +1,145 @@
+/*-------------------------- slider functonality --------------------------*/
+var slideIndex = 1;
+var showSlides = (n) => {
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    var dots = document.getElementsByClassName("dot");
+    if (n > slides.length) { slideIndex = 1 }
+    if (n < 1) { slideIndex = slides.length }
+    for (i = 0; i < slides.length; i++) {

```

Comment:
use a variable to get the length out of the loop and use the variable in here

Reply:
done